spec/models/scimitar/lists/query_parser_spec.rb

```diff
@@ -405,27 +405,27 @@
       query = @instance.to_activerecord_query(MockUser.all)
 
       expect(query.count).to eql(1)
-      expect(query.pluck(:id)).to eql([user_1.id])
+      expect(query.pluck(:primary_key)).to eql([user_1.primary_key])
```

Comment:
Note that I could just as well write `user_1.id`, since ActiveRecord provides that alias given that the model underneath _has_ declared its alternative primary key name. I wanted to maintain as much clarity as possible in the test suite as possible, though, so have updated references to `MockUser#id` -> `MockUser#primary_key`wherever I noticed them.